python/ray/serve/_private/deployment_state.py

```diff
@@ -1187,14 +1190,16 @@ def check_stopped(self) -> bool:
 
         timeout_passed = time.time() >= self._shutdown_deadline
         if timeout_passed:
-            # Graceful period passed, kill it forcefully.
-            # This will be called repeatedly until the replica shuts down.
-            logger.info(
-                f"{self.replica_id} did not shut down after grace "
-                "period, force-killing it. "
-            )
+            if not self._logged_shutdown_message:
+                logger.info(
+                    f"{self.replica_id} did not shut down after grace "
+                    "period, force-killing it. "
+                )
```

Comment:
if we're guarding this anyways can we also guard it with a `not RAY_SERVE_DISABLE_SHUTTING_DOWN_INGRESS_REPLICAS_FORCEFULLY`? right now we print the following log lines one right after the other which is kind of confusing

```
replica did not shut down after grace period, force-killing it.
replica did not shut down because it had not finished draining requests. Going to wait until the draining is complete. You can force-stop the replica by setting RAY_SERVE_DISABLE_SHUTTING_DOWN_INGRESS_REPLICAS_FORCEFULLY to 0.
```

Reply:
makes sense